### PR TITLE
Improving arabic translation

### PR DESCRIPTION
**Pull request by BoFFire**
Improving arabic translation

### CODE REVIEW

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -55,6 +61,7 @@
     <string name="action_compose">حرر</string>
     <string name="action_login">التسجيل بواسطة ماستدون</string>
     <string name="action_logout">خروج</string>
+    <string name="action_logout_confirm">متأكد مِن أنك تود الخروج من الحساب %1$s ؟</string>
```
Comment (connyduck) [inline app/src/main/res/values-ar/strings.xml]: you can not split up the 1%$s, it is one placeholder for the username

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -132,6 +141,7 @@
     <string name="pref_title_notification_settings">الاشعارات</string>
     <string name="pref_title_edit_notification_settings">تعديل الاشعارات</string>
     <string name="pref_title_notifications_enabled">الإخطارات</string>
+    <string name="pref_summary_notifications">للحساب %1$s</string>
```
Comment (connyduck) [inline app/src/main/res/values-ar/strings.xml]: you can not split up the 1%$s, it is one placeholder for the username

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -233,16 +252,30 @@
...
     <string name="load_more_placeholder_text">حمِّل المزيد</string>
 
+    <string name="add_account_name">إضافة حساب</string>
+    <string name="add_account_description">إضافة حساب ماستدون جديد</string>
+
+    <string name="action_lists">القوائم</string>
+    <string name="title_lists">القوائم</string>
+    <string name="title_list_timeline">الخط الزمني للقائمة</string>
+
+    <string name="compose_active_account_description">النشر بواسطة حساب %1$s</string>
```
Comment (connyduck) [inline app/src/main/res/values-ar/strings.xml]: you can not split up the 1%$s, it is one placeholder for the username

**Comment by BoFFire** [comment]
Hello,
Can you suscribe to my own repo on transifex to get that fucking file ?
Thank you.

https://www.transifex.com/opendz/tusky-in-arabic/dashboard/

Best regards,

**Comment by BoFFire** [comment]
Hello,
Can you suscribe to my own repo on transifex to get that fucking file ?
Thank you.

https://www.transifex.com/opendz/tusky-in-arabic/dashboard/

Best regards,

**Comment by BoFFire** [comment]
Hi !
I'm sorry, I don't have a computer. I made all the translation via my phone on Transifex, I think that the error happened when I was trying to copy & paste.
If you are interested, please come to get that file from : https://www.transifex.com/opendz/tusky-in-arabic/

Best regards,

**Comment by BoFFire** [comment]
Hi,
File as extracted from transifex : https://cloud.indie.host/s/Whhj431sxcQZQvN

**Comment by connyduck** [comment]
Yes, but can you tell me what the translations for `action_logout_confirm`, `compose_active_account_description`, `pref_summary_notifications` with one %1$s placeholder are, please?

**Comment by BoFFire** [comment]
Hi,
Consider them untranslated if these strings cause any issues when packaging or delete the lines.
Right To Left languages (RTL) are somewhat difficult to handle some variables that appear weired for a non RTL person.
So, delete them ^-^

**Comment by BoFFire** [comment]
Closed pull request for cleaning.

**Comment by connyduck** [comment]
Oh I am sorry, it seems it this was a problem with github having troubles with right-to-left languages. Translations look fine when I check the PR out.

**Comment by BoFFire** [comment]
Hi,
Pushing new strings today in 1 hour :))

**Pull request merged**